lib/gql_serializer.rb

```diff
@@ -98,6 +98,13 @@ def self.serialize(record, hasharray, options)
     hash
   end
 
+  def self.coerce_value(value)
+    return value.to_f if value.is_a? BigDecimal
+    return value.new_offset(0).strftime("%FT%TZ") if value.is_a? DateTime
+    return value.utc.iso8601 if value.is_a? Time
```

Comment:
I'm adding support for Time. It wasn't working for the type that's returned in our app.